### PR TITLE
Update to OpenTelemetry Collector v0.13.0

### PR DESCRIPTION
**Pull request by jpkrohling**
As part of the update to use the OpenTelemetry Collector v0.13.0:

- we removed the auth processor from this repo, in favor of the native one
- aligned the versions in the manifests
- use the latest builder, with support for 'replaces'

﻿Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -2,12 +2,15 @@ dist:
     module: github.com/observatorium/observatorium-otelcol
     name: observatorium-otelcol
     description: Observatorium OpenTelemetry Collector Distribution
-    version: 0.1.0
+    version: 0.1.1
```
Comment (objectiser) [inline manifest.yaml:5]: nit: would 0.2.0 be more appropriate given the change in components (e.g. auth processor)

File: manifest.yaml [hunk 2]
```diff
@@ -2,12 +2,15 @@ dist:
...
 processors:
-  - gomod: github.com/observatorium/observatorium-otelcol/processors/authenticationprocessor v0.0.0
-    path: ./processors/authenticationprocessor
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor v0.10.0
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.10.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sprocessor v0.13.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.13.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor v0.13.0
+
+replaces:
```
Comment (objectiser) [inline manifest.yaml:14]: Could you add a comment against all/each line here to explain why they are necessary, referencing any issues - so they can be cleaned up with the relevant issue is fixed?

**Review by objectiser — APPROVED** [review]
LGTM - just a few minor comments.

File: manifest.yaml (hunk 1 above)
Comment (jpkrohling) [reply manifest.yaml]: Done!

File: manifest.yaml (hunk 2 above)
Comment (jpkrohling) [reply manifest.yaml:14]: Done! Is the comment sufficient?

Comment (objectiser) [reply manifest.yaml:14]: Is there a github issue related to these? or are they going to stay this way?
Otherwise they are good.

Comment (jpkrohling) [reply manifest.yaml:14]: There's a general one: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1217

Comment (objectiser) [reply manifest.yaml:14]: ok thanks

**Pull request merged**